src/Sentry.NLog/SentryTarget.cs

```diff
@@ -438,14 +448,28 @@ private User GetUser(LogEventInfo logEvent)
                 return null;
             }
 
-            return new User
+            var user = new User
             {
                 Email = User.Email?.Render(logEvent),
                 Id = User.Id?.Render(logEvent),
                 IpAddress = User.IpAddress?.Render(logEvent),
                 Username = User.Username?.Render(logEvent),
-                Other = User.Other.ToDictionary(a => a.Name, b => b.Layout.Render(logEvent)),
             };
+
+            if (User.Other?.Count > 0)
```

Comment:
This change is unrelated to the logger changes, right?
Ideally in the future it would land on a separate PR please

Reply:
I just reacted to warning about unnecessary allocations and delegate capture.

Reply:
But now you have also received a unit-test that verifies the Other-output actually works :)